pkg/templates/templates_generated.go

```diff
@@ -1629,6 +1646,29 @@ installCrictl() {
     fi
     rm -rf ${CRICTL_DOWNLOAD_DIR}
 }
+{{if TeleportEnabled}}
+downloadTeleportdPlugin() {
+    TELEPORTD_VERSION=$1
+    if [[ -z ${TELEPORTD_PLUGIN_DOWNLOAD_URL} ]]; then
```

Comment:
nit: can we pass in the download url as a parameter too

Also do we have any default value that we can use if RP doesn't provide an overriding?